media-libs/avbin-bin/avbin-bin-8.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="A thin wrapper around FFmpeg"
+HOMEPAGE="http://avbin.github.io/"
+SRC_URI="
+	amd64? ( mirror://github/AVbin/AVbin/avbin-linux-x86-64-v${PV}.tar.bz2 )
+	x86? ( mirror://github/AVbin/AVbin/avbin-linux-x86-32-v${PV}.tar.bz2 )"
+
+LICENSE="GPL-3 LGPL-3"
```

Comment:
Why don't you add the source based ebuild? GPL requires that the source need be accessible. [TLDR legal](https://tldrlegal.com/license/gnu-general-public-license-v3-(gpl-3))